guides/doc-Upgrading_Project/topics/proc_upgrading-a-connected-project-server.adoc

```diff
@@ -33,12 +33,26 @@ include::../common/modules/snip_warning-maintain-config-noop.adoc[]
 # {foreman-installer} --foreman-proxy-dns-managed=false \
 --foreman-proxy-dhcp-managed=false
 ----
-ifdef::katello[]
-. Optional: If you use PostgreSQL as an external database, install the `postgresql-evr` package on the PostgreSQL server:
+ifdef::katello,satellite,orcharino[]
```

Comment:
@AkshayGadhaveRH Need your help here. If installing `postgresql-contrib` applies to Satellite, Katello, and Orcharhino, why the changes of this PR are not visible for Orcharhino? You can see the bot-generated list of HTML files.

I saw the `master.adoc` but there is no condition at all for this file.